cli/ts/genProofs.ts

```diff
@@ -336,23 +360,34 @@ const genProofs = async (args: any) => {
         return 
     }
 
+    let maciState: MaciState
     // Build an off-chain representation of the MACI contract using data in the contract storage
-
-    // some rpc endpoint like bsc chain has limitation to retreive history logs
-    let fromBlock = 0
-    const txHash = args.transaction_hash
-    if (txHash) {
-        const txn = await signer.provider.getTransaction(txHash);
-        fromBlock = txn.blockNumber
+    if (args.state_file) {
+        // @todo actually read the file first 
+        const content = JSON.parse(fs.readFileSync(args.state_file, 'utf-8').toString())
```

Comment:
This assumes that the input file format is correct. It's important to handle potential exceptions that could arise from incorrect file formats. Adding error handling for the JSON parsing process could prevent runtime errors and improve the robustness of the script.